src/patternfly/components/Form/styles.scss

```diff
@@ -199,178 +149,48 @@ $pf-c-form__select--m-error-svg--Viewbox: "0 0 26 15.8" !default;
-    }
-  }
-
-  // Select & datalist
-  &__select,
-  &__datalist {
-    padding-right: var(--pf-c-form__select--PaddingRight);
-
-    /* stylelint-disable */
-    -webkit-appearance: none;
-    -moz-appearance: none;
-    -ms-appearance: none;
-    /* stylelint-enable */
-
-    &:not([multiple]) {
-      @include pf-bg-svg($pf-c-form__select--chevron-svg--Coordinates, $pf-c-form__select--chevron-svg--Color, $pf-c-form__select--chevron-svg--Width, $pf-c-form__select--chevron-svg--Height);
-    }
-
-    &.pf-m-disabled,
-    &:disabled {
-      color: var(--pf-c-form__input--disabled--Color);
-      background-color: var(--pf-c-form__input--disabled--BackgroundColor);
-      border-color: var(--pf-c-form__input--disabled--BorderColor);
-      @include pf-bg-svg($pf-c-form__select--chevron-svg--Coordinates, $pf-c-form__select--chevron-svg--disabled--Color, $pf-c-form__select--chevron-svg--Width, $pf-c-form__select--chevron-svg--Height);
-    }
-
-    &:hover::-webkit-calendar-picker-indicator,
-    &:focus::-webkit-calendar-picker-indicator {
-      pointer-events: none;
-      opacity: 0;
-    }
-
-    &:-moz-focusring {
-      color: transparent;
-    }
-  }
-
-  // Datalist
-  &__datalist {
-
-    // Disabled
-    &.pf-m-disabled,
-    &:disabled {
-      cursor: not-allowed;
-    }
-  }
-
-  // Input, checkbox, radio, select
-  &__checkbox,
-  &__radio,
-  &__select {
-    cursor: pointer;
-
-    // Disabled
-    &:disabled,
-    &.pf-m-disabled {
-      cursor: not-allowed;
-    }
-  }
-
-  // Checkbox and radio container
-  &__check {
-    // Handle additional elements within __check
-    display: flex;
-    flex: 1;
-    align-items: baseline;
-
-    // Space additional elements within __check
-    > *:not(.pf-c-form__label):not(.pf-c-form__checkbox):not(.pf-c-form__radio) {
-      margin-left: var(--pf-c-form__check--addons--MarginLeft);
-    }
-
-    .pf-c-form__label {
-      // Remove default label padding
-      padding-bottom: 0;
-
-      &:not(.pf-m-disabled):hover {
-        cursor: pointer;
-      }
-
-      &.pf-m-disabled:hover {
-        cursor: not-allowed;
-      }
-    }
   }
 
-  // Checkbox, radio
-  &__checkbox,
-  &__radio {
-    margin-right: var(--pf-c-form__checkbox--MarginRight);
+  // // Checkbox and radio container
+  // &__check {
+  //   // Handle additional elements within check
+  //   display: flex;
+  //   flex: 1;
+  //   align-items: baseline;
+
+  //   // Space additional elements within pf-c-check
+  //   > *:not(.pf-c-form__label):not(.pf-c-check) {
+  //     margin-left: var(--pf-c-form__check--addons--MarginLeft);
+  //   }
+
+  //   .pf-c-form__label {
+  //     // Remove default label padding
+  //     padding-bottom: 0;
+
+  //     &:not(.pf-m-disabled):hover {
+  //       cursor: pointer;
+  //     }
+
+  //     &.pf-m-disabled:hover {
+  //       cursor: not-allowed;
+  //     }
+  //   }
+  // }
+
+  // // Checkbox, radio
+  .pf-c-check {
+    margin-right: var(--pf-c-check--MarginRight);
```

Comment:
This style gets overridden by the following:
```
.pf-c-form .pf-m-inline>* {
    flex: auto 0;
    margin-right: var(--pf-c-form--inline--MarginRight);
}
```
(copied from inspector)

Reply:
ah ya, this was an issue with the handlebars modifier again :(  They aren't name spaced but I think we will need to do it in this case to avoid this in the future.